tuna/worker_interface.py

```diff
@@ -89,54 +65,49 @@ def __init__(self, **kwargs):
 
     allowed_keys = set([
         'machine', 'gpu_id', 'num_procs', 'barred', 'bar_lock', 'envmt',
-        'reset_interval', 'fin_steps', 'fin_infile', 'fin_outfile', 'job_queue',
-        'queue_lock', 'label', 'fetch_state', 'docker_name', 'end_jobs',
-        'config_type', 'dynamic_solvers_only', 'session_id'
+        'reset_interval', 'fin_steps', 'job_queue', 'queue_lock', 'label',
+        'fetch_state', 'docker_name', 'end_jobs', 'config_type',
+        'dynamic_solvers_only', 'session_id'
     ])
     self.__dict__.update((key, None) for key in allowed_keys)
 
-    #for pylint
+    #system vars
     self.machine = None
+    #multiprocess vars
     self.gpu_id = None
     self.num_procs = None
     self.barred = None
     self.bar_lock = Lock()
-    self.envmt = []
-    self.fin_steps = []
-    self.fin_infile = None
-    self.fin_outfile = None
     self.job_queue = None
     self.queue_lock = Lock()
-    self.is_fdb = False
+    self.end_jobs = None
+    #job detail vars
+    self.envmt = []
+    self.fin_steps = []
     self.fetch_state = ['new']
-    self.compile_only = False
     self.docker_name = TUNA_DOCKER_NAME
-    self.gpu = None
     self.label = None
-    self.end_jobs = None
-    self.solver_id_map, _ = get_solver_ids()
-    self.id_solver_map, _ = get_id_solvers()
     self.dynamic_solvers_only = False
-    self.config_type = ConfigType.convolution if self.config_type is None else self.config_type
-    self.config_dict = None
     self.session_id = None
 
     self.__dict__.update(
         (key, value) for key, value in kwargs.items() if key in allowed_keys)
 
+    #initialize tables
+    self.config_type = ConfigType.convolution if self.config_type is None else self.config_type
     self.dbt = DBTables(session_id=self.session_id,
```

Comment:
should we set DBTables in the child class? then we could move ConfigType into the miopen part as well. We could set self.dbt = self.set_db_tables() and then define this function in the miopen class.

Reply:
Sure, that makes sense enough. I'll adapt that in the next PR.